pulp_2to3_migration/app/plugin/deb/migrator.py

```diff
@@ -51,6 +59,12 @@ class DebMigrator(Pulp2to3PluginMigrator):
     future_types = {
         'deb': pulp_2to3_models.Pulp2DebPackage,
```

Comment:
I found that adding my new metadata types here changes things. (I no longer run into an error, but I don't get exactly the result I want either.)

This gives me something I can continue investigating for now.